examples/tutorials/rigid_object_tutorial.py

```diff
@@ -1,44 +1,83 @@
-# [setup]
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.4.2
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %%
+# !curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | bash -s
+
+
+# %%
+# %run -i /content/habitat-sim/examples/colab_utils/load_env.py
+
+# %%
+# %cd /content/habitat-sim
 import math
-import os
-import random
+import os, sys
 
 import cv2
-import magnum as mn
+import imageio
+try:
+  import magnum as mn
+except ImportError:
+  if "google.colab" in sys.modules:
+    conda_path = '/usr/local/lib/python3.6/site-packages/'
+    user_path = '/root/.local/lib/python3.6/site-packages/'
+    os.environ['IMAGEIO_FFMPEG_EXE'] = '/usr/bin/ffmpeg'
+    sys.path.insert(0, conda_path)
+    sys.path.insert(0, user_path)
+  import magnum as mn #If it STILL doesn't work, try restarting your runtime.
 import numpy as np
 
 import habitat_sim
 import habitat_sim.utils.common as ut
 
-dir_path = os.path.dirname(os.path.realpath(__file__))
-data_path = os.path.join(dir_path, "../../data")
-output_path = os.path.join(dir_path, "rigid_object_tutorial_output/")
+import git
+import time
+import random
 
+repo = git.Repo('.', search_parent_directories=True)
+dir_path = repo.working_tree_dir
+data_path = os.path.join(dir_path, "data")
+output_path = os.path.join(dir_path, "examples/tutorials/rigid_object_tutorial_output/")
 
+#TODO, we should display the videos in matplotlib so that they work locally and on collab
 def make_video_cv2(observations, prefix="", open_vid=True, multi_obs=False):
```

Comment:
Agreed, I'd like to see an observation->video utility function. That said, there are many options here (e.g. separate videos for each sensor, video matrices, embedding one observation into another, etc...). It may be tough to design a single function to handle all of these cases. 

Maybe we can have a couple of fairly configurable choices available using the same basic framework? Better for these to live in the viz utilities. Can come as a follow-up PR also 👍

Reply:
Agree that it should be a follow up PR